### ISSUE TITLE
In iOS, not able to open .epub and .mobi file using file opener plugin

### ISSUE THREAD

**Issue by Topiya** [pwlin/cordova-plugin-file-opener2#223]
Hello,

I want to open .epub and .mobi files using file-opener plugin of Ionic-cordova in iOS app. But while doing this I am getting below error.

`can not handle uti`

Can anyone please help me to solve this error?

**Comment by godoyrw** [comment]
See this... [https://github.com/pwlin/cordova-plugin-file-opener2/issues/2](url)

**Comment by Topiya** [comment]
@godoyrw Can you please recheck the given URL because it is not working.

**Comment by godoyrw** [comment]
It's ok now! Sorry!

**Comment by shnist** [comment]
@Topiya did the suggestions in that issue resolve your issue?

**Comment by Topiya** [comment]
@shnist No, It didn't solve my issue. Is there any other workaround for this issue?

**Comment by shnist** [comment]
@Topiya can you provide a code snippet from your project? This will help diagnose the issue. One thing you can try, if you haven't done so already, is to try and provide the relevant MIME type (https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types). So for example, an .epub file would look something like:

```javascript
cordova.plugins.fileOpener2.open('file.epub',  'application/epub+zip', {  error : function(){ },   success : function(){ } });
```

**Comment by prasadpandari** [comment]
Anyone able to find a solution for this. I have same issue.
  cordova.plugins.fileOpener2.open(
    dirEntry.nativeURL + fileName,
    mimetype,
    {
      error: (e) => {
        console.log(`Error status: ${e.status} - Error message: ${e.message}`);
      },
      success: () => {
      })

Iam getting "Error status: 9 - Error message: Could not handle UTI

**Comment by Pawel86400** [comment]
Any idea what can cause this error and how to solve it?

My MIME type is `application/pdf`. 
Surprisingly only 1 user out of many is experiencing this issue on iPhone 7 Plus.

Unfortunately I'm not able to reproduce it on other devices or in the simulator.
